docs/tutorials/get_started_with_qiskit.ipynb

```diff
@@ -0,0 +1,285 @@
+      ],
+      "text/plain": [
+       "<Figure size 700x500 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.visualization import plot_histogram\n",
+    "\n",
+    "plot_histogram(\n",
+    "    job.result().quasi_dists\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77054d4e",
+   "metadata": {},
+   "source": [
+    "And that's it! You now have everything you need to start writing and running quantum programs."
```

Comment:
Would it make sense to discuss where to go next? Maybe simply linking to the tutorials index at https://qiskit.org/documentation/tutorials/index.html?

Reply:
You mean https://qiskit.org/documentation/tutorials.html?

Reply:
Ah yeah, I did. Thanks.